copulas/univariate/selection.py

```diff
@@ -1,21 +1,15 @@
 import numpy as np
 
+from scipy.stats import kstest
 from copulas import get_instance
 
-
-def ks_statistic(model, X):
```

Comment:
This was my bad, I didn't realize scipy had an implementation when I implemented, let's use it.